src/services/websockets.js

```diff
@@ -20,26 +20,32 @@ export function initialise (actionCreatorBinder) {
   } = actionCreatorBinder({...SearchResultActions, ...TagActions});
   primus.on('data', function received (data) {
     console.log('incoming socket data', data);
-    if (data.connection) {
-      saveSocketConnectionId(data.connection);
+    if (data.graphql) {
+      saveSearchResult(data);
+    }
+  });
+
+  primus.once('open', (data) => {
```

Comment:
For what it's worth, this is one of the things that primuc [claim to be a major selling point of their module](https://github.com/primus/primus#advantages)